### PR TITLE
Fix/6355 long event description hides adress

### PR DESCRIPTION
**Pull request by marcussc**
## Description
With long texts the Description and Adress, the Adress would not be displayed. 

To fix this we now have an added addressTextBox. 

If long texts in Adress or Description run out of space now, only the to long text gets cut off. And booth are displayed. 

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6355

## Screenshots
Screen Shot 2021-04-12 at 14 57 30

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift [hunk 1]
```diff
@@ -303,17 +303,17 @@ class DMPosterGenerationViewController: UIViewController, UITextFieldDelegate {
 		let pdfView = PDFView()
 		let pdfDocument = PDFDocument(data: templateData.template)
 
-		guard let qrCodeImage = viewModel.traceLocation.qrCode(size: CGSize(width: qrCodeSideLengthField.cgFloatValue, height: qrCodeSideLengthField.cgFloatValue), qrCodeErrorCorrectionLevel: .medium) else { return pdfView }
-		guard let hexColorString = descriptionFontColorField.text else { return pdfView }
+//		guard let qrCodeImage = viewModel.traceLocation.qrCode(size: CGSize(width: qrCodeSideLengthField.cgFloatValue, height: qrCodeSideLengthField.cgFloatValue), qrCodeErrorCorrectionLevel: .medium) else { return pdfView }
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift:306]: Can this be removed?

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift [hunk 2]
```diff
@@ -303,17 +303,17 @@ class DMPosterGenerationViewController: UIViewController, UITextFieldDelegate {
...
 
-		try? pdfDocument?.embedImageAndText(
-			image: qrCodeImage,
-			at: CGPoint(x: qrCodeOffsetXField.cgFloatValue, y: qrCodeOffsetYField.cgFloatValue),
-			text: viewModel.posterDescription,
-			of: descriptionFontSizeField.cgFloatValue,
-			and: UIColor().hexStringToUIColor(hex: hexColorString),
-			with: CGRect(x: descriptionOffsetXField.cgFloatValue, y: descriptionOffsetYField.cgFloatValue, width: descriptionWidthField.cgFloatValue, height: descriptionHeightField.cgFloatValue)
-		)
+//		try? pdfDocument?.embedImageAndText(
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift:309]: Will we create a separate ticket to update the Developer Menu according to the new changes?

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift (hunk 1 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift]: Removed 🤙

File: src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift (hunk 2 above)
Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift]: Issue: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6373

Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Developer Menu/Features/DMPosterGeneration/DMPosterGenerationViewController.swift]: Will take care of it.

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**